### ISSUE TITLE
tag namespacing for tags txt file

### ISSUE THREAD

**Issue by deltarayedge** [Nandaka/DanbooruDownloader#132]
Is it possible to write the namespace for tags, i.e. "copyright:love live"  to the tags text file instead of just the tag?

**Comment by Nandaka** [comment]
I don't think that is supported by the site, as the program just pass the whole tag to query.

**Comment by CuddleBear92** [comment]
actually its all in there as classes, but i guess you use the api so thats different.
on the html code it lists each namespace with numbers.

category-0 = unnamespaced
category-1 = artist = creator
category-3 = copyrights = series
category-4 = character


other downloaders that might use html like hydrus does use those very well to get the tags in the right namespace. 

but while we are on namespaces, many of the sites are different for many use cases. 
allowing the user to choose what namespace to come out at the end of say the tag text file got alot of say.

many things creator and series is better for many reasons.
but on other sites there is many more namespaces too.

**Comment by Nandaka** [comment]
yap, each sites use different category/namespace.

**Comment by CuddleBear92** [comment]
well what you should do then it load the namespaces for the user and have a default set namespace the program comes with, and then let the use choose what do actually use.

its only a very very few that uses the category numbers like these.
most of the sites use tag-type[namespace] for its site. 
most of the time these are all the same across the board with a few sites having more than others with its classes

we already handled this on the Hydrus front with our presets of classes to namespaces. 

you can feel free to take a look at the namespaces we have presets for in my github.
they are just yaml files that is meant to work with Hydrus as a downloader. 
but as i said, most of the sites use tag-type in some form or another and should be easy to fix.

but its important to give the user choice in what namespace to be exported.

https://github.com/CuddleBear92/Hydrus-Presets-and-Scripts/tree/master/Downloader%20Presets
